packages/stepper/src/StepperContext.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {
+  createContext,
+  ReactNode,
+  useContext,
+  useState,
+  useEffect,
+} from 'react';
+import { createUniqueClassName } from '@leafygreen-ui/lib';
+import { StepperContextValues } from './types';
+
+export const StepperContext = createContext({});
+export const useStepperContext = () =>
+  useContext<Partial<StepperContextValues>>(StepperContext);
+
+const StepperContextProvider = ({
+  darkMode,
+  children,
+}: {
+  darkMode: boolean;
+  children: ReactNode;
+}) => {
+  // TODO: This dark mode state should ideally be managed in a global state.
+  const [isDarkMode, setIsDarkMode] = useState<boolean>(darkMode);
+  const stepIconClassName = createUniqueClassName();
+  const stepLabelClassName = createUniqueClassName();
```

Comment:
These classnames don't really need to be in `context`. They are static values that can just be exported from a constants file. As written we need to generate a new classname for each stepper